### PR TITLE
[PACKAGE] Add Python Interpreter in MUSIC Application

### PR DESCRIPTION
**Pull request by juliencastelneau**
Linked to [PR #669](https://github.com/Inria-Asclepios/music/pull/669)

Creation of a script **python_package.sh** in package stuffs

1. create a virtual env
2. install requirements in env (itk, sitk, tensorflow, learning models,...)
3. create an archive
4. copy it in package

### CODE REVIEW

File: packaging/python/python_packager.sh.in [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash 
+
+for i in $*; do
+	echo $i
```
Comment (mathildemerle) [inline packaging/python/python_packager.sh.in]: could be removed from the final PR?

File: packaging/apple/mac_packager.sh.in [hunk 2]
```diff
@@ -76,6 +80,7 @@ do
   install_name_tool -change $musicPluginsBuildDir/libmscPipelinesPlugin.dylib @executable_path/../PlugIns/libmscPipelinesPlugin.dylib $f
 done
 
+
```
Comment (mathildemerle) [inline packaging/apple/mac_packager.sh.in:80]: you can remove this empty line

File: packaging/python/python_packager.sh.in [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	# Set some plugin dirs
+	if [[ $i = *"musicPlugins"* ]]; then
+		musicPythonInterpreterDir="$i/medPipeline/python/interpreter"
+	fi
+done
+echo $musicPythonInterpreterDir
```
Comment (mathildemerle) [inline packaging/python/python_packager.sh.in]: can be remove too

**Comment by mathildemerle** [comment]
Is this code working on Windows? ^^'

**Comment by juliencastelneau** [comment]
> Is this code working on Windows? ^^'

I am not sure I understand the question.
Is there the ability to choose a python interpreter on Windows ?
The answer is no.
Is the project compiled on Windows with this PR ?
The answer is Yes

**Review by mathildemerle — APPROVED** [review]

**Review by paulineMig — APPROVED** [review]

**Review by mjuhoor — APPROVED** [review]

**Pull request merged**